src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java

```diff
@@ -274,10 +252,9 @@ public void configMotors() {
 		armMotor1.setInverted(false);
 		armMotor1.getEncoder().setPosition(0);
 
+		enableShoulderLimits();
 		armMotor1.enableSoftLimit(CANSparkMax.SoftLimitDirection.kForward, true);
 		armMotor1.enableSoftLimit(CANSparkMax.SoftLimitDirection.kReverse, true);
-		armMotor1.setSoftLimit(CANSparkMax.SoftLimitDirection.kForward, ARM_FORWARD_LIMIT);
-		armMotor1.setSoftLimit(CANSparkMax.SoftLimitDirection.kReverse, ARM_REVERSE_LIMIT);
```

Comment:
This needs to be added back, either here or in `enableShoulderLimits()`, since `enableShoulderLimits()` currently only enables the soft limits but doesn't set them.